DirectEmployers/MyJobs
# Generate fake JobViews and put them into MemSQL.

## What's this PR do?
Generates fake job views and adds them to MemSQL. Useful for testing analytics locally, since all QC MemSQL requires VPN access.

I tried to create a useful distribution of cities/regions/countries/job titles/etc, and my results thus far suggest that I was successful. 

Data is random and will not represent true job views. The foundations for the data were pulled from actual job views, but should have any company-identifying information stripped.
 
## Screenshots (if appropriate)
![screen shot 2017-02-27 at 8 09 50 am](https://cloud.githubusercontent.com/assets/2728248/23362528/26adbe8c-fcc4-11e6-84ec-966ff5fd9086.png)

## Where should the reviewer start?
With a solid understanding of what a job view is, and opinions on what fake data should look like.

## How should this be manually tested?
Try running the command (`python manage.py make_data` or `dkm make_data` if you're using Docker) against your local MemSQL db. Confirm that you are satisfied with what you see in analytics.

## Any background context you want to provide?
No.
